### ISSUE TITLE
Few issues

### ISSUE THREAD

**Issue by itsho** [davdiv/DecryptSQLite#1]
Hi.

Thank you for your cool project.
I've used it and it really helped me 👍 

I have some minor issues and suggestions:
1. Please use Nuget instead of referencing a (missing) DLL.
2. Make the Main method [STAThread].
3. When possible, use English messages for people who can't read French. 😉
4. Add a SLN file.
5. A suggestion - make it into a CLI tool (without UI).

a side note - I've tested it with `System.Data.SQLite` version `1.0.109.0` and it works great

Thanks again

**Comment by davdiv** [comment]
Hello,

Thank you for opening this issue. I am glad that this small tool helped you!
It is true that this tool could be improved a lot, thank you for your good suggestions!
However, I don't really have time to work on it, it is just a small utility that I did when I needed it, and I do not use it very often anymore.
I noticed that you have done a similar tool at https://github.com/itsho/RemoveSqlitePassword That's nice! Well done!
Apparently you live in Israel! That is great! I went to Israel last month and I really enjoyed the trip!
God bless you!
Have a good day!
